style.css

```diff
@@ -6,42 +6,65 @@
 
 @font-face {
   font-family: 'Lucida Sans Italic';
-  src: url('lucidasansitalic.ttf') format('truetype');
+  src: url('assets/lucidasansitalic.ttf') format('truetype');
```

Comment:
You have fonts in `assets` and `SegoeUI`? One place, please, some `fonts` folder